portfolio/src/main/webapp/index.html

```diff
@@ -40,7 +40,49 @@ <h1>Hi, I'm Keely!</h1>
         </div>
       </div>
     </section>
-    <section>
+    <section id="about-me">
+      <div>
+        <h2>~ more about me ~</h2>
+        <div class="list-item">
+          <div class="img-container">
+            <img src="/images/abby1.jpg" class="about-img">
+            <img src="/images/abby2.jpg" class="about-img">
+            <img src="/images/abby3.jpg" class="about-img">
+          </div>
+          <div class="description-container">
+            I have an 8-year old golden retriever named Abby! She's the laziest dog I've ever met, but she'll do
+            anything for a treat and a belly rub. She's also the world's worst guard dog; she'll shower any intruder
+            with hugs and kisses.
+          </div>
+        </div>
+        <div class="list-item">
+          <div class="description-container">
+            I'm an aspiring baker (what I really mean is that I'm terrible at baking). My goal is to perfect a
+            banana bread recipe by the end of the summer. Sounds simple, but I still haven't succeeded. My
+            greatest "baking" accomplishment came from Pillsbury Ready-to-Bake sugar cookies which are to die for.
```

Comment:
Let's talk about baking sometime! :)

Reply:
Definitely - I'll be asking for lots of advice!